### ISSUE TITLE
web admin: unable to set type=dhcp-listener for an interface

### ISSUE THREAD

**Issue by nqb** [inverse-inc/packetfence#5512]
**Describe the bug**
When I tried to set `type=dhcp-listener` for an ethernet interface without any other setting, I got following error message on frontend: 

```
Unable to validate
type: 'dhcp-listener' is not a valid value
```

**To Reproduce**
1. Go to Interfaces menu
2. Click on an interface without IP address
3. Try to set type to "DHCP listener"

**Expected behavior**
Frontend should allow you to set `type=dhcp-listener` for an interface.

**Additional context**
No difference if I specified an IP address on the interface.

I'm able to set `type=dhcp-listener` by editing directly `pf.conf` and checkup doesn't report any issue.

**Comment by jrouzierinverse** [comment]
Which version? Unable to replicate in devel.

**Comment by nqb** [comment]
I tested on devel and tested again this morning: I'm able to reproduce.

My setup:
- My first interface is eth0
- Configuration in `/etc/sysconfig/network-scripts/ifcfg-eth0`:
```
DEVICE=eth0
HWADDR=
ONBOOT=yes
BOOTPROTO=static
NM_CONTROLLED=no
IPADDR=192.168.121.48
NETMASK=255.255.255.0
```
- I try to edit interface from web admin and got same error message.

JSON payload sent:
```json
{
    "additional_listening_daemons": [],
    "address": "192.168.121.48/24",
    "coa": null,
    "dhcpd_enabled": null,
    "dns": null,
    "high_availability": 0,
    "hwaddr": "52:54:00:bb:2e:8a",
    "id": "eth0",
    "ifindex": "2",
    "ipaddress": "192.168.121.48",
    "ipv6_address": null,
    "ipv6_prefix": null,
    "is_running": true,
    "master": null,
    "name": "eth0",
    "nat_enabled": null,
    "netmask": "255.255.255.0",
    "network": "192.168.121.0",
    "network_iseditable": false,
    "networks": [],
    "reg_network": null,
    "split_network": null,
    "type": "dhcp-listener",
    "vip": null,
    "vlan": null
},
```

**Comment by jrouzierinverse** [comment]
This was caused by the fix for #5466 .
Question for @julsemaan or @fdurand is 'dhcp-listener' a valid type?
Or should it be we only see it in options_additional_listening_daemons?

**Comment by julsemaan** [comment]
dhcp-listener is a valid type

**Comment by jrouzierinverse** [comment]
Where would it go in this list?
https://github.com/inverse-inc/packetfence/blob/devel/html/pfappserver/lib/pfappserver/Model/Enforcement.pm#L26

**Comment by jrouzierinverse** [comment]
Fixed by e7f520129df58ee39ad4d8e2147adcddd23bbceb
